lib/glific/flags.ex

```diff
@@ -38,9 +39,18 @@ defmodule Glific.Flags do
   end
 
   defp out_of_office_check do
-    {:ok, now} = DateTime.now(@timezone)
+    timezone = Partners.organization().timezone
+
+    {:ok, now} =
```

Comment:
@anshulrr I am merging this PR but I think this can be improve further. 

We will look into this in the new PR